src/core/drive/visit.ts

```diff
@@ -50,7 +50,8 @@ export type VisitOptions = {
   restorationIdentifier?: string
   shouldCacheSnapshot: boolean
   frame?: string
-  acceptsStreamResponse: boolean
+  acceptsStreamResponse: boolean,
+  initiator?: HTMLAnchorElement | HTMLFormElement
```

Comment:
Could we declare this property without the `?` optional modifier? `VisitOption` instances are typically passed around as `Partial<VisitOption>` types, so the `Partial` makes it optional.

Reply:
Without `?` TypeScript moans because it's not present in `defaultOptions`:

```
Property 'initiator' is missing in type '{ action: "advance"; historyChanged: false; visitCachedSnapshot: () => void; willRender: true; updateHistory: true; shouldCacheSnapshot: true; acceptsStreamResponse: false; }' but required in type 'VisitOptions'.
```

Reply:
Could its default value be `document.documentElement`, since that's what it's currently dispatched from?

Reply:
I'm not strongly opposed to this, but just wonder if having an undefined `initiator` usefully signals that the `Visit` is a programmatic one?

Reply:
> I'm not strongly opposed to this, but just wonder if having an undefined initiator usefully signals that the Visit is a programmatic one?

I suppose it doesn't _really_ matter since a programmatic visit could be determined if `visit.initiator === document.documentElement`

Reply:
Or if we want a default, perhaps `initiator` is the wrong name for this? i.e. for programmatic visits the HTML element doesn't "initiate" the visit, the event just gets dispatched on it. It's not as clear, but perhaps `Visit#element` would work?

Reply:
Default initiator implemented in: 36dd97bdb653dbc2c600729ad1364907d94455dc